udata/core/spatial/api.py

```diff
@@ -85,7 +85,7 @@ def get(self):
     location='args', default=25)
 
 
-@ns.route('/zones/<pathlist:ids>/', endpoint='zones')
```

Comment:
As discussed, we should make sure zones don't have `/`?

Reply:
On dev there is none:
```
>>> GeoZone.objects(id__contains='/')
[]
```